### DISCUSSION TITLE
Set a cooldown or delay before additional button press?

### DISCUSSION THREAD

**Discussion by Crazyairborne** [AntiMicroX/antimicrox#400] [Q&A]
So Im looking for a way to make it so a button cannot be pressed again before a set number of seconds, lets say 5 seconds. Is there a way to implement this into this program? I tried adding a delay to the end of the macro, but if i press the button repeatedly, it fires, even if the "delay" from the previous button press is still active. Looking for any ideas. Thanks! 

Background: Im using this application to trigger a lap timer for minibikes, using a laser sensor. I need to make the button press have a short cool down so it only trips one time as the minbike passes the laser.

**Comment by pktiuk** [comment]
No, there is no such an option.
